src/super_gradients/training/transforms/transforms.py

```diff
@@ -608,10 +629,37 @@ def __call__(self, sample: dict) -> dict:
                 ar_thr=self.ar_thr,
                 border_value=self.border_value,
             )
-            sample["image"] = img
-            sample["target"] = target
+
+            sample.image = img
+            sample.is_crowd = np.array([0] * len(targets) + [1] * len(crowd_targets), dtype=bool)
+            sample.bboxes_xywh = np.concatenate([targets[:, 0:4], crowd_targets[:, 0:4]], axis=0, dtype=sample.bboxes_xywh.dtype)
+            sample.labels = np.concatenate([targets[:, 4], crowd_targets[:, 4]], axis=0, dtype=sample.labels.dtype)
         return sample
 
+    def get_equivalent_preprocessing(self):
+        raise NotImplementedError("get_equivalent_preprocessing is not implemented for non-deterministic transforms.")
+
+    # def __call__(self, sample: dict) -> dict:
```

Comment:
Don't forget to remove/de-comment (depends on what we decide regarding __call__(), see earlier comments).